interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -237,6 +268,13 @@ namespace cling {
       return;
     }
 
+    // When not using C++ modules, we now have a PCH and we can safely setup
+    // our callbacks without fearing that they get ovewritten by clang code.
+    // The modules setup is handled above.
+    if (!usingCxxModules) {
```

Comment:
Can that be an else to the `if (cxxmodules)` case?

Reply:
Well, the whole point is (as described in the comment) to have it before/after the `Initialize` call which overwrites the PCH in the non-module case.

Reply:
Ah, that's right. Let's see if the bots are green and I will merge it in.